src/main/scala/com/fulcrumgenomics/util/miseq/Sample.scala

```diff
@@ -26,13 +26,15 @@
   */
 package com.fulcrumgenomics.util.miseq
```

Comment:
Perhaps we should move this out of `miseq` into a `com.fulcrumgenomics.illumina` package?